### PR TITLE
[1.4] Remove trailing whitespace

### PR DESCRIPTION
**Pull request by ExterminatorX99**
Trailing whitespace is evil.
I'm good, so I killed it.

### CODE REVIEW

File: patches/Terraria/Terraria/Main.cs.patch [hunk 1]
```diff
@@ -481,22 +481,22 @@
...
-@@ -49869,6 +_,8 @@
- 			else if (Terraria.Graphics.Effects.Filters.Scene["MoonLordShake"].IsActive()) {
+@@ -49870,6 +_,8 @@
  				Terraria.Graphics.Effects.Filters.Scene.Deactivate("MoonLordShake");
  			}
-+			
-+			SkipMoonLordEffects:
  
++			SkipMoonLordEffects:
++
```
Comment (steviegt6) [inline patches/Terraria/Terraria/Main.cs.patch:496]: Your modification to this patch here removed the newline above this label but added a newline below it, unsure of if that was intentional.
It also caused the differ to add an extra line of patch context.

**Review by steviegt6 — CHANGES_REQUESTED** [review]
A small concern.

File: patches/Terraria/Terraria/Main.cs.patch (hunk 1 above)
Comment (ExterminatorX99) [reply patches/Terraria/Terraria/Main.cs.patch:496]: I don't see a problem.
Both patches result in
```cs
// Empty line
SkipMoonLordEffects:
// Empty line
```

Comment (ExterminatorX99) [reply patches/Terraria/Terraria/Main.cs.patch:496]: And the context line was just changed. There is one less at the top, but one extra at the bottom

**Pull request merged**